src/amo/components/Addon/index.js

```diff
@@ -38,6 +39,7 @@ import {
   ENABLED,
   INSTALL_SOURCE_DETAIL_PAGE,
   UNKNOWN,
+  INCOMPATIBLE_NOT_FIREFOX,
```

Comment:
Nitpick: we try to sort everything alphabetically.

Reply:
ah..ok..good to know, thanks

Reply:
It took me some time to remember that, but @tofumatt traumatized me I guess because that's one of the first things I check now 😄